docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -15,18 +15,22 @@ Show the version number. `bigchaindb -v` does the same thing.
 
 ## bigchaindb configure
 
-Generate a local config file (which can be used to set some or all [BigchainDB node configuration settings](configuration.html)). It will auto-generate a public-private keypair and then ask you for the values of other configuration settings. If you press Enter for a value, it will use the default value.
+Generate a local configuration file (which can be used to set some or all [BigchainDB node configuration settings](configuration.html)). It will auto-generate a public-private keypair and then ask you for the values of other configuration settings. If you press Enter for a value, it will use the default value.
+
+Since BigchainDB supports multiple databases you need to always specify the
+database backend that you want to use. At this point only two database backends
+are support, `rethinkdb` and `mongodb`.
 
 If you use the `-c` command-line option, it will generate the file at the specified path:
 ```text
-bigchaindb -c path/to/new_config.json configure
+bigchaindb -c path/to/new_config.json configure rethinkdb
```

Comment:
Is the name of the database backend _required_, or is there a default one? If it is required, then that should be mentioned.